code/game/objects/structures/petrified_statue.dm

```diff
@@ -5,13 +5,20 @@
 	density = TRUE
 	anchored = TRUE
 	max_integrity = 200
+	// Should we leave a brain behind when the statue is wrecked?
+	var/brain = TRUE
 	var/timer = 480 //eventually the person will be freed
 	var/mob/living/petrified_mob
 
-/obj/structure/statue/petrified/Initialize(mapload, mob/living/L, statue_timer)
+/obj/structure/statue/petrified/relaymove()
+	return
+
+/obj/structure/statue/petrified/Initialize(mapload, mob/living/L, statue_timer, save_brain)
```

Comment:
You should have defaults set in case no args are put in

Reply:
The args are only ever used IF they are put in. If they aren't, it just uses the default variable on the statue.
The Args override this default if provided.

Reply:
I was saying this more as readability it would be better if defaults were set here and overridden when necessary since that’s where we are expecting these to be set over adding subtypes, but it’s fine.